x/wasm/keeper/proposal_handler.go

```diff
@@ -107,9 +107,7 @@ func handleMigrateProposal(ctx sdk.Context, k types.ContractOpsKeeper, p types.M
 	if err != nil {
 		return sdkerrors.Wrap(err, "contract")
 	}
-	if err != nil {
-		return sdkerrors.Wrap(err, "run as address")
-	}
+
```

Comment:
If a function returns an error, that error must be checked.

Reply:
Function indicates `sdk.AccAddressFromBech32`?
If so, wouldn't the error check above suffice?
(https://github.com/CosmWasm/wasmd/pull/905/files#diff-a5c1aab521b573562de4814c157ce3c9b7564672c24f0acafffdb003de9c16fbR107-R109)

Reply:
Ah, there are duplicate error checks here — all good 👍